websites/Y/Yandex Music/dist/metadata.json

```diff
@@ -0,0 +1,22 @@
+{
+  "$schema":"https://schemas.premid.app/metadata/1.0",
+  "author":{
+    "name":"D3rise",
+    "id":"253913831896645632"
+  },
+  "url":"music.yandex.ru",
+  "description":{
+    "ru":"Стриминговый сервис компании «Яндекс», позволяющий прослушивать музыкальные композиции, альбомы, подборки музыкальных треков и получать персональные рекомендации.",
+    "en":"Yandex streaming service that allows you to listen to music tracks, albums, and collections of music tracks and get personal recommendations."
+  },
+  "service":"Yandex Music",
+  "version": "1.0.0",
+  "logo":"https://music.yandex.ru/blocks/meta/i/og-image.png",
+  "thumbnail":"https://music.yandex.com/pages/main/i/og/home_com.png",
```

Comment:
The images provided should be hosted in [Imgur](https://imgur.com) or on a CDN that gives you permission to change image files.